From 14f951ca2d354ad8fd068d0f3d8648d5c0d1e60e Mon Sep 17 00:00:00 2001
From: Myy Miouyouyou <myy@miouyouyou.fr>
Date: Thu, 19 Oct 2017 22:40:26 +0200
Subject: [PATCH 23/28] ARM: DTS: rk3288-tinker.dts: Defining the SPI interface

Taken from, and tested by @TonyMac32 .

Well, the original one was tested by him but I had to adapt the
registers definitions to the new 64-bits LPAE-compliant syntax.

Therefore that *might* break, along with a few other patches.

Signed-off-by: Myy Miouyouyou <myy@miouyouyou.fr>
---
 arch/arm/boot/dts/rk3288-tinker.dts | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-tinker.dts b/arch/arm/boot/dts/rk3288-tinker.dts
index 6f4c0843..f4b4525c 100644
--- a/arch/arm/boot/dts/rk3288-tinker.dts
+++ b/arch/arm/boot/dts/rk3288-tinker.dts
@@ -471,6 +471,24 @@
 
 };
 
+&spi2 {
+	status = "okay";
+	max-freq = <50000000>;
+	spidev@0 {
+		compatible = "rockchip,spi_tinker";
+		reg = <0x0 0>;
+		spi-max-frequency = <50000000>;
+		spi-cpha = <1>;
+	};
+	spidev@1 {
+		compatible = "rockchip,spi_tinker";
+		reg = <0x0 1>;
+		spi-max-frequency = <50000000>;
+		spi-cpha = <1>;
+	};
+};
+
+
 &pinctrl {
 	pcfg_pull_none_drv_8ma: pcfg-pull-none-drv-8ma {
 		drive-strength = <8>;
-- 
2.11.0

